### PR TITLE
docs: add minutes for 2020-06-30 initiatives meeting

### LINKED ISSUE #116: Frontend Initiatives Meeting : June 30, 2020
# Frontend Initiatives Meeting

**When** : Tuesday, June 30, 2020 @ 15:00 CET
**Where** : remote, let me know if you didn't get a calendar invite

# Agenda
- Proposal Ratification
  - [PeerDeps for Singletons](https://github.com/dhis2/notes/issues/67)
  - [PeerDeps for wrapped external dependencies](https://github.com/dhis2/notes/issues/67)
  - [Release Strategy](https://github.com/dhis2/notes/issues/101) - slower, avoid major releases where possible, not **complete** until propagated
  - [Jira vs. GH Issues](https://dhis2.slack.com/archives/CDX5CE15W/p1592488981276700)
  - Managing workload and context switching (see below)

### PR DESCRIPTION
**Pull request by amcgee**
Closes #116

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: decisions/2020/06/30-frontend-peerDeps-and-releases.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+* [Meeting Recording](https://drive.google.com/file/d/1CPkBMWMy38iNu62K6FgiVYvANxmEw6mg/view)
+
+# Notes
+
+## Good News - Developer Workshop
+
+- Successfully hosted first online DHIS2 application developer workshop last week!
+- 30+ participants from 19 countries
+- Find
```
Comment (varl) [inline decisions/2020/06/30-frontend-peerDeps-and-releases.md:28]: What does this mean?

File: decisions/2020/06/30-frontend-peerDeps-and-releases.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+(DHIS2 account required)
+
+* [Meeting Recording](https://drive.google.com/file/d/1CPkBMWMy38iNu62K6FgiVYvANxmEw6mg/view)
+
+# Notes
+
+## Good News - Developer Workshop
+
+- Successfully hosted first online DHIS2 application developer workshop last week!
```
Comment (varl) [inline decisions/2020/06/30-frontend-peerDeps-and-releases.md:26]: :clap::clap::clap:

File: decisions/2020/06/30-frontend-peerDeps-and-releases.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+- 30+ participants from 19 countries
+- Find 
+- Huge thanks to Jan-Gerke, Hendrik, Christos, Milagros, Lars, and Wahed as well as Matthieu, Alice, and Grant for making this possible!
+- Some room for improvement for
```
Comment (varl) [inline decisions/2020/06/30-frontend-peerDeps-and-releases.md:30]: Extra "for" at the end?

File: decisions/2020/06/30-frontend-peerDeps-and-releases.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+**Proposal**: Leave as-is for now, wrap and re-export dependencies from libraries that are clear extensions of an external library.
+However, plan a formal investigation into the advantages and challenges of moving to peer dependencies for a later major release (across libraries)
+
+**Status**: Approved
+
+**Next steps**: Plan investigation into implications of moving to peer dependencies in a future major version
+
+### PeerDependencies for Singletons
+
+Notably applies to `react`, `react-dom`, `styled-jsx`, `@dhis2/app-runtime`, `@dhis2/d2-i18n` (until deprecated), `@dhis2/ui` or `@dhis2/ui-core`.
```
Comment (varl) [inline decisions/2020/06/30-frontend-peerDeps-and-releases.md:47]: Are `dhis2/ui` and children singletons? Especially confused about the core library; that should just be a collection of components. 🤔

Comment (ismay) [reply decisions/2020/06/30-frontend-peerDeps-and-releases.md:47]: I'm not sure if ui has to be a singleton, but ui-core needs to be for the components in it that rely on react context. Unless we remove react context from ui-core, that'd also resolve it.

**Review by Mohammer5 — APPROVED** [review]
+ what @varl found

File: decisions/2020/06/30-frontend-peerDeps-and-releases.md (hunk 4 above)
Comment (varl) [reply decisions/2020/06/30-frontend-peerDeps-and-releases.md:47]: Ah! That'd be it. Thanks for that reminder, @ismay.

Comment (ismay) [reply decisions/2020/06/30-frontend-peerDeps-and-releases.md:47]: N.p. 😊👍

File: decisions/2020/06/30-frontend-peerDeps-and-releases.md (hunk 1 above)
Comment (amcgee) [reply decisions/2020/06/30-frontend-peerDeps-and-releases.md:28]: Typo

**Pull request merged**